src/main/java/org/opentripplanner/model/plan/pagecursor/PageCursorFactoryParameters.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner.model.plan.pagecursor;
+
+import java.time.Instant;
+
+public interface PageCursorFactoryParameters {
```

Comment:
Needs JavaDoc on interface and methods. Does not need a lot, purpose of interface and what is the context of "kept" and "removed" -> ItineraryFilter.

Reply:
Done.